library/ssl_tls.c

```diff
@@ -6998,14 +6998,14 @@ int mbedtls_ssl_set_calc_verify_md( mbedtls_ssl_context *ssl, int md )
 {
 #if defined(MBEDTLS_SSL_PROTO_TLS1_2)
     if( ssl->minor_ver != MBEDTLS_SSL_MINOR_VERSION_3 )
-        return MBEDTLS_ERR_SSL_INVALID_VERIFY_HASH;
+        return( 1 );
```

Comment:
I know this doesn't change anything it practice, but I think returning `-1` would be slightly clearer. Error codes tend to be negative, and we already use `-1` is several places as a generic error code.

Reply:
Changing.